research/auto-fed-rl/README.md

```diff
@@ -0,0 +1,72 @@
+# Auto-FedRL
+
+Components for running the algorithm proposed in
+["Auto-FedRL: Federated Hyperparameter Optimization for Multi-institutional Medical Image Segmentation"](https://arxiv.org/abs/2203.06338) (ECCV 2022)
+with [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html).
+
+You can find the original experimental implementation [here](https://github.com/guopengf/Auto-FedRL).
+
+> **Note:** For more examples of FL algorithms with the CIFAR-10 dataset,
+> see [here](../../examples/cifar10).
+
+## License
+
+The code in this directory is released under Apache v2 License.
+
+## 1. Set PYTHONPATH
+Set `PYTHONPATH` to include custom files of this example:
+
+We use both the utils from the [CIFAR-10 examples](../../examples/cifar10)
+and files under [./src](./src):
+```
+export PYTHONPATH=${PWD}/../../examples/advanced/cifar10:${PWD}/../../examples/advanced/cifar10/pt/utils:${PWD}/src
+```
```

Comment:
shall we specify which utils are reused here to help users (cifar10_data_splitter and network I think)?